tests/test_examples.py

```diff
@@ -9,10 +9,13 @@
 from glob import glob
```

Comment:
should not be part of solver commit...

Reply:
This is always a bit tricky, because I already made separate PR for some of these unrelated issues you see. So once that PR is merged (#651 ) then this diff should disppear. Plus, I am in part already relying on these changes (in this case, running all examples to ensure correctness of the new solver).

Any preference if I should in the meantime remove these unrelated changes untill they are merged back in from master?

Reply:
I've reverted these changes on this branch